docs/README.md

```diff
@@ -1,149 +1,77 @@
-
-## Configure the CLI to use an HTTPS proxy
-
-The CLI can be setup to use an HTTPS proxy. To setup an HTTPS proxy, an environment variable called `HTTPS_PROXY` must
- be created. The variable must be set to the address of the HTTPS proxy, and its port using the following format:
-`{PROXY IP}:{PROXY PORT}`.
-
-## Using OpenWhisk from an iOS app
-
-You can use OpenWhisk from your iOS mobile app or Apple Watch by using the OpenWhisk iOS SDK. For more details, refer to the [iOS documentation](./mobile_sdk.md).
-
-## Using REST APIs with OpenWhisk
-
-After your OpenWhisk environment is enabled, you can use OpenWhisk with your web apps or mobile apps with REST API calls.
-For more details about the APIs for actions, activations, packages, rules, and triggers, see the [OpenWhisk API documentation](http://petstore.swagger.io/?url=https://raw.githubusercontent.com/openwhisk/openwhisk/master/core/controller/src/main/resources/whiskswagger.json).
-
-## OpenWhisk Hello World example
-To get started with OpenWhisk, try the following JavaScript code example.
-
-```
-/**
- * Hello world as an OpenWhisk action.
- */
-function main(params) {
-    var name = params.name || 'World';
-    return {payload:  'Hello, ' + name + '!'};
-}
-```
-
-To use this example, follow these steps:
-
-1. Save the code to a file. For example, *hello.js*.
-
-2. From the OpenWhisk CLI command line, create the action by entering this command:
-
-    ```
-    $ wsk action create hello hello.js
-    ```
-
-3. Then, invoke the action by entering the following commands.
-
-    ```
-    $ wsk action invoke hello --blocking --result
-    ```
-
-    This command outputs:
-
-    ```
-    {
-        "payload": "Hello, World!"
-    }
-    ```
-
-    ```
-    $ wsk action invoke hello --blocking --result --param name Fred
-    ```
-
-    This command outputs:
-
-    ```
-    {
-        "payload": "Hello, Fred!"
-    }
-    ```
-
-You can also use the event-driven capabilities in OpenWhisk to invoke this action in response to events. Follow the [alarm service example](./packages.md#creating-and-using-trigger-feeds) to configure an event source to invoke the `hello` action every time a periodic event is generated.
-
-
-## System details
-
-You can find additional information about OpenWhisk in the following topics:
-
-* [Entity names](./reference.md#openwhisk-entities)
-* [Action semantics](./reference.md#action-semantics)
-* [Limits](./reference.md#system-limits)
-* [REST API](./reference.md#rest-api)
+OpenWhisk is an [Apache Incubator Project](https://incubator.apache.org/projects/openwhisk.html). It is an open source implementation of a distributed, event-driven compute service. You can run it on your own hardware on-prem, or in the cloud. When running in the cloud you could use PaaS version of the OpenWhisk provided by IBM Bluemix cloud, or you can provision it yourself into Bluemix IaaS, or other IaaS clouds, such as Amazon AWS, Microsoft Azure, Google GCP, etc. 
```

Comment:
use PaaS -> use a PaaS (although PaaS should be defined.)
IBM Bluemix cloud -> IBM Bluemix